package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I think it's confusing to add a `package.json` even though we don't publish this as an npm package, nor do we install any packages from npm. Could we avoid adding it right now, and just have `skylint.sh` in place of `npm run skylint`?

Also, if there's no CI enforcement included in this PR, then we'll probably introduce more lint violations. Have you thought about adding to the circleci config?